### PR TITLE
Added input params, votingPhaseTime, executionPhaseTime info to proposals stored in db and fixed error showing empty data on some proposals

### LINKED ISSUE #71: No data on latest proposals
Even the chain data is indexed to the latest, there is no data for the last few proposals.

image

### PR DESCRIPTION
**Pull request by MonikaCat**
Fixes #71

### CODE REVIEW

File: imports/api/governance/server/methods.ts [hunk 1]
```diff
@@ -33,198 +34,201 @@ function mergeObjects(object1, object2, object3) {
...
-                                $set: proposalData.proposal[element],
-                            }
+    }
 
-                        )
-                    } catch (e) {
-                        console.log(e);
+        Object.keys(proposalData).forEach(function (element) {
+            try {
+                Proposals.upsert(
```
Comment (kwunyeung) [inline imports/api/governance/server/methods.ts]: Consider upserting with mongodb bulk operations in a loop

**Review by kwunyeung — APPROVED** [review]
Some of the logics can be broken down into smaller functions which will make future maintenance easier.

File: imports/api/governance/server/methods.ts (hunk 1 above)
Comment (MonikaCat) [reply imports/api/governance/server/methods.ts]: Ok! I have applied those changes to the code :)

**Review by kwunyeung — APPROVED** [review]
LGTM

**Pull request merged**